sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosDiagnosticsTest.java

```diff
@@ -224,6 +224,14 @@ public void supplementalResponseStatisticsList() throws Exception {
         supplementalResponseStatisticsListNode = (ArrayNode) jsonNode.get("supplementalResponseStatisticsList");
         assertThat(storeResponseStatistics.size()).isEqualTo(7);
         assertThat(supplementalResponseStatisticsListNode.size()).isEqualTo(7);
+
+        //verifying all components from StoreResponseStatistics
+        for(JsonNode node : supplementalResponseStatisticsListNode) {
+            assertThat(node.get("storeResult")).isNotNull();
+            assertThat(node.get("requestResponseTimeUTC")).isNotNull();
```

Comment:
Should there be validation to check if it is in the correct format?

Reply:
Added